Tests/RegressionTests.cs

```diff
@@ -150,21 +150,21 @@ private static TestCaseData[] GetRegressionTestParameters()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "155.210%"},
+                {"Compounding Annual Return", "191.086%"},
```

Comment:
Have reverted this change for now. I see what you are getting at: existing new order calls that supply a decimal will expect the implicit conversion but currently there is a behaviour change. I was hoping to avoid this, but I think it's necessary to add methods such as:

LimitOrderFractional(symbol, quantity, limitPrice)

...or to add overloads that cause retention of the decimal fragments:

LimitOrder(symbol, quantity, limitPrice, string tag = null, bool isQuantityFractional = false)